io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -1,9 +1,9 @@
 mdt_file: 'net.mdt'
-host_ip: x.xx.xxx.xx
-peer_ip: x.xx.xxx.xy
+host_public_ip: x.xx.xxx.xx
```

Comment:
That's a no because ssh happens through Public interfaces and that's how the commands on the peer gets executed. If we consider test interfaces with Public IP's, the script will change the IP for them due to net_id parameter in bpt file. This will fail ssh to the peer. 
So we'll have to make sure that while running HTX, we have a separate interface set aside which has a Public IP and that should be left untouched.